README.md

```diff
@@ -29,6 +29,14 @@ joining our [public Slack channel.](https://adhocteam-public.herokuapp.com)
 We think a sign of a good developer is one who asks questions sooner, rather
 than later. We are happy to help guide you to the right solution.
 
+On Overengineering
```

Comment:
Maybe "On Overengineering vs. Underengineering"? The flipside is that I do tend to take off points (say from 5 to 4) for things that have the right answer but don't seem production ready because there are too many magic variables and whatnot.